### DISCUSSION TITLE
Add "Overwrite or Add" prompt when importing

### DISCUSSION THREAD

**Discussion by leonisir** [outline/outline#5739] [Ideas]
Could you please add a new feature for this? 
and import api only used for importing, Why not also use it to update document

**Comment by tommoor** [comment]
I'm not really clear how this would be used, to update a document you need it's ID

**Reply by leonisir** [reply]
When importing in the same collection under the same file path，
such as import 1.md . It is much friendlier to have an import in the same path
collection/
└── document
    └── 1.md
    └── 1.md

another issue，import api only have parentDocumentId，maybe can add a DocumentId to importing as update
![image](https://github.com/outline/outline/assets/35453327/6404752d-a2f2-44c4-a49a-49fd88476ad9)
